### PR TITLE
[Task CS-9] Database Setup

### PR DESCRIPTION
**Pull request by jmysliv**

### CODE REVIEW

**Comment by twrdyyy** [comment]
@jmysliv fix pylint errors plox

**Review by skusnierz — CHANGES_REQUESTED** [review]

File: requirements.txt [hunk 1]
```diff
@@ -1,2 +1,4 @@
-pytest
 flask
+pytest
```
[the remark this answers is not in the document]
Comment (jmysliv) [reply requirements.txt:2]: @twrdyyy tak kazał, też mu to mówiłem

[the remark this answers is not in the document]
Comment (skusnierz) [reply requirements.txt:2]: @jmysliv can you please take that project seriously for once? You seem to be joking right now and this is a serious task

File: entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#! /bin/bash
+
+echo "Installing requirements..."
```
[the remark this answers is not in the document]
Comment (jmysliv) [reply entrypoint.sh:3]: no

**Review by twistzzmc — APPROVED** [review]

**Review by skusnierz — APPROVED** [review]

**Pull request merged**